packages/components/src/components/popover/popover.jsx

```diff
@@ -194,7 +194,7 @@ const Popover = ({ ...props }) => {
                     <div
                         className={classNames('dc-popover', className)}
                         id={id}
-                        onMouseEnter={toggleOpen}
+                        onMouseOver={toggleOpen}
                         onMouseLeave={toggleClose}
```

Comment:
`onMouseOut` is the corresponding exit for `onMouseOver`.  So should this be `onMouseOut` instead of `onMouseLeave`?